Makefile

```diff
@@ -239,7 +239,7 @@ endif
 
 shellcheck:
 	@echo "🐚 📋 Linting shell scripts with shellcheck"
-	$(SHELLCHECK) $(find -name '*.sh' -type f | grep -v "venv/\|git/\|.pytest_cache/\|htmlcov/")
+	$(SHELLCHECK) $(shell find -name '*.sh' -type f | grep -v 'venv/\|git/\|.pytest_cache/\|htmlcov/')
```

Comment:
How about searching for all shell files (we can add exclude list similarly to .venv example):
```shell
$ find . -type f ! -path "*.venv/*" | xargs file -i - | grep "text/x-shellscript" | awk -F":" '{print $1}'
```

Reply:
I tested and this command find less files than the one I am using (it did not find `_test/conftest.sh`)

Reply:
That is correct. The `_test/conftest.sh` is not an shell file. Look at the first line:
```
#!/usr/bin/env bats
```

So it's bats interpreted file not shell and as such shellcheck should not be ran against it.

Reply:
I think that is not a problem https://github.com/koalaman/shellcheck/issues/417

or do you want to only check bash/sh files?

Reply:
I would be more careful and check only bash/sh files. There can be some list of *additional* files to be checked if they are not proper  `"text/x-shellscript"` type.

Reply:
I remove the bats file from the shellcheck with grep (to have a simpler command), but added a note here https://github.com/konveyor/pelorus/issues/825 to fix it later